### PR TITLE
ENH: Use CIFTI for carpetplot when available

### LINKED ISSUE #2026: Add CIFTI visualization to report
Something ala #2025 to quickly catch problems

### PR DESCRIPTION
**Pull request by mgxd**
Closes #2026 

Uses BOLD CIFTI file to generate carpetplot when available
Depends on nipreps/niworkflows#491

### CODE REVIEW

**Comment by mgxd** [comment]
report with: https://12731-53175327-gh.circle-artifacts.com/0/tmp/ds005/derivatives/fmriprep/sub-01.html

report without: https://12734-53175327-gh.circle-artifacts.com/0/tmp/ds054/derivatives/fmriprep/sub-100185.html

**Comment by effigies** [comment]
No colors for the CIFTI carpetplot?

**Comment by mgxd** [comment]
The colorbars on the CIFTI plot are not being properly displayed in firefox, though it appears fine on chromium based browsers. I'm looking into this now..

File: fmriprep/data/reports-spec.yml [hunk 1]
```diff
@@ -108,10 +108,10 @@ sections:
       the white-matter (WM) and within cerebro-spinal fluid (CSF) show the mean BOLD
       signal in their corresponding masks. DVARS and FD show the standardized DVARS
       and framewise-displacement measures for each time point.<br />A carpet plot
-      shows the time series for all voxels within the brain mask. Voxels are grouped
-      into cortical (blue), and subcortical (orange) gray matter, cerebellum (green)
-      and white matter and CSF (red), indicated by the color map on the left-hand
-      side.
+      shows the time series for all voxels within the brain mask, or if CIFTI output
```
Comment (oesteban) [inline fmriprep/data/reports-spec.yml:111]:
```suggestion
      shows the time series for all voxels within the brain mask, or if ``--cifti-output``
```

**Comment by mgxd** [comment]
@effigies rendering on chromium / firefox 👍

**Comment by effigies** [comment]
Same here. Thanks. (Sorry for not seeing this sooner.)

I'm not a huge fan of having conditional language in the text, as opposed to changing the text based on what format is available, but that can be a new issue. Might be a good issue for new contributors.

File: setup.cfg [hunk 2]
```diff
@@ -25,7 +25,7 @@ install_requires =
     nibabel ~= 3.0
     nipype ~= 1.4
     nitime
-    niworkflows >= 1.2.0rc1,<1.3
+    niworkflows @ git+https://github.com/mgxd/niworkflows.git@03e8b7886e434b986749a0746214d64a1b55b41d
```
Comment (mgxd) [inline setup.cfg:28]:
```suggestion
    niworkflows >= 1.2.0rc2,<1.3
```

**Pull request merged**